share/functions/fish_default_key_bindings.fish

```diff
@@ -13,7 +13,7 @@ function fish_default_key_bindings -d "Default (Emacs-like) key bindings for fis
     end
 
     # This is the default binding, i.e. the one used if no other binding matches
-    bind $argv "" self-insert
+    bind $argv \uE000 self-insert
```

Comment:
There are several problems with this change but this one is a deal breaker. We absolutely do not want to introduce the use of Unicode private-use chars in our scripts. Not least because we hope to eliminate their use entirely in the not too distant future. We also do not want to introduce magic constants that do not have an associated symbolic name. Finally, the unicode private-use chars fish currently reserves are defined in *src/common.h* (search for RESERVED_CHAR_BASE).